src/css/wui.css

```diff
@@ -1,4 +1,26 @@
+.absolute-left {
```

Comment:
These css classes are not namespaced and I fear they would produce conflicts in any page in which they are used.